docusaurus/docs/cms/admin-panel-customization/homepage.md

```diff
@@ -0,0 +1,761 @@
+---
+title: Homepage customization
+description: Learn about the Strapi admin panel Homepage and how to customize it with widgets.
+toc_max_heading_level: 6
+tags:
+- admin panel
+- homepage
+- widgets
+- features
+---
+
+# Homepage customization
+
+The <Icon name="house" /> Homepage is the landing page of the Strapi admin panel. By default, it provides an overview of your content with 2 default widgets:
+
+- _Last edited entries_: Displays recently modified content entries, including their content type, status, and when they were updated.
+- _Last published entries_: Shows recently published content entries, allowing you to quickly access and manage your published content.
+
+<ThemedImage
+  alt="Homepage with default widgets"
+  sources={{
+    light: '/img/assets/admin-homepage/admin-panel-homepage.png',
+    dark: '/img/assets/admin-homepage/admin-panel-homepage_DARK.png',
+  }}
+/>
+
+These default widgets cannot currently be removed, but you can customize the Homepage by creating your own widgets.
+
+:::note
+If you recently created a Strapi project, the Homepage may also display a quick tour above widgets if you haven't skipped it yet.
+:::
+
+## Adding custom widgets
+
+To add a custom widget, you need to:
+
+- register it
+- create a widget component
+
+<!-- ### Enabling the feature flag
+
+To enable custom widgets, set the `unstableWidgetsApi` feature flag to `true` in [the `config/features` file](/cms/configurations/features):
+
+<Tabs groupId="js-ts">
+
+<TabItem value="js" label="JavaScript">
+
+```js title="config/features.js"
+module.exports = ({ env }) => ({
+  future: {
+    unstableWidgetsApi: true,
+  },
+});
+```
+
+</TabItem>
+
+<TabItem value="ts" label="TypeScript">
+
+```ts title="config/features.ts"
+export default ({ env }) => ({
+  future: {
+    unstableWidgetsApi: true,
+  },
+});
+```
+
+</TabItem>
+
+</Tabs>
+
+After enabling the feature flag and restarting your application, the Homepage will be able to display the registered custom widgets. -->
+
+### Registering custom widgets
+
+To register a widget, use `app.widgets.register()`:
+- in the plugin’s [`register` lifecycle method](/cms/plugins-development/server-api#register) of the `index` file if you're building a plugin (recommended way),
+- or in the [application's global `register()` lifecycle method](/cms/configurations/functions#register) if you're adding the widget to just one Strapi application without a plugin.
+
+<Tabs groupId="js-ts">
+<TabItem value="javascript" label="JavaScript">
+
+```jsx title="src/plugins/my-plugin/admin/src/index.js"
+import pluginId from './pluginId';
+import MyWidgetIcon from './components/MyWidgetIcon';
+
+export default {
+  register(app) {
+    // Register the plugin itself
+    app.registerPlugin({
+      id: pluginId,
+      name: 'My Plugin',
+    });
+    
+    // Register a widget for the Homepage
+    app.widgets.register({
```

Comment:
Also worth a callout: this API only works with Strapi 5.13 and above. Trying to call this with older versions of Strapi will crash the admin.

So plugin developers who want to register plugins should either:
- set `^5.13.0` as their `@strapi/strapi` peerDependency in their plugin package.json. This peer dependency is what powers the marketplace's compatibility check. 
- check if the API exists before calling it:
  ```js
  if ('widgets' in app) {
    // proceed with the registration
  }
  ```

I would recommend the 1st approach if the whole purpose of the plugin is to register plugins. The second makes more sense if a plugin wants to add a widget but most of its functionality is elsewhere